### DISCUSSION TITLE
ui.text pre-formated

### DISCUSSION THREAD

**Discussion by Rmfjd1978** [h2oai/wave#1365] [Ideas]
It was a good option to add a boolean parameter to ui.text that indicates that the content is pre-formated (with the default value to false).
I sugest this because the alternative is to put on the content parameter the usage of the html tag `<pre>` and with this usage the css style is not the same.

**Comment by mturoci** [comment]
If you want to keep the text styling the same as the rest of the text, use

```py
q.page['example'] = ui.form_card(
    box='1 1 4 10',
    items=[ui.text('<span style="white-space: pre">Text         aaa</span>')]
)
```
